docs/docs/guides/testing.md

```diff
@@ -49,26 +49,26 @@ export default Form;
 #### Test
 
 ```jsx
-import { RecoilRoot } from 'recoil';
-import { fireEvent, render, screen } from '@testing-library/react';
+import {RecoilRoot} from 'recoil';
```

Comment:
Just out of curiosity. Why not adding spaces around objects? I knew the rule about not leaving spaces around array contents, but I thought leaving spaces between object contents and the curly braces was acceptable.

Reply:
Yeah, spaces there are pretty common with open source projects.  However, the internal Facebook prettier standard has no spaces.  So, this change is just for consistency with the rest of the code base and samples.  No value judgement either way.

Reply:
Thanks. I didn't know 🙂